sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerReadSchemaTransformProvider.java

```diff
@@ -76,40 +76,34 @@ public String description() {
         + "\n"
         + "Example configuration for performing a read using a SQL query: ::\n"
         + "\n"
-        + "    pipeline:\n"
-        + "      transforms:\n"
-        + "        - type: ReadFromSpanner\n"
-        + "          config:\n"
-        + "            instance_id: 'my-instance-id'\n"
-        + "            database_id: 'my-database'\n"
-        + "            query: 'SELECT * FROM table'\n"
+        + "    - type: ReadFromSpanner\n"
+        + "      config:\n"
+        + "        instance_id: 'my-instance-id'\n"
+        + "        database_id: 'my-database'\n"
+        + "        query: 'SELECT * FROM table'\n"
```

Comment:
This is consistent with the other providers, so I just added it to this PR since it is the only other Java provider that did not follow this pattern. In there future, I'll make sure to separate into multiple PRs